File: ide/idelsp/symbolresolve/testdata_rs/app/service.rs
use crate::utils::text;

pub struct Service;

impl Service {
    pub fn new() -> Service {
        Service
    }
}

// Defined but never referenced as service::boot; resolves through the
// definitions phase.
pub fn boot() {
    let _ = text::slugify("boot");
}

pub fn run() {
    helper();
}

fn helper() {}

File: ide/idelsp/symbolresolve/testdata_zig/app/service.zig
const text = @import("../utils/text.zig");

pub const Service = struct {
    pub fn init() Service {
        return .{};
    }
};

// Defined but never referenced as service.boot; resolves through the
// definitions phase.
pub fn boot() void {
    _ = text.slugify("boot");
}

pub fn run() void {
    helper();
}

fn helper() void {}
